src/NuGet.Indexing/DisposableAction.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
```

Comment:
nit: newline after copyright

Reply:
This was pulled in via dependency before?

Reply:
Yes, NuGet.Core.

Reply:
Added newline.